openhtf/plugs/usb/__init__.py

```diff
@@ -52,10 +58,18 @@ def _open_usb_handle(**kwargs):
   Returns:
     Instance of UsbHandle.
   """
-  if conf.Config().usb_server:
-    return None
-  else:
-    return local_usb.LibUsbHandle.Open(**kwargs)
+
+  serial = None
+  remote_usb = conf.Config().remote_usb.strip()
+  if (remote_usb == 'EtherSync'):
```

Comment:
We used to have a decently elegant pattern where we would register a class name to a config value. Just a note that we probably want to do something like that here eventually, but this if-test is fine for now, especially since EtherSync is currently our only remote USB option.